### DISCUSSION TITLE
Work on Mali

### DISCUSSION THREAD

**Discussion by Elijah313** [skyline-emu/skyline#2076] [Ideas]
I really appreciate your efforts to improve this emulator. I wish you every success. I'm really waiting for an update Mali and thank you for everything 🙏.I apologize if I'm wrong, I'm using google translate, please understand me

**Comment by Mononono87** [comment]
Mali is worse at emulation that snapdragon,also it's not on the developers priority list right now.
Also without drivers being on mali it gives it a huge disadvantage

**Reply by Elijah313** [reply]
Thank you for replying to me, I appreciate your words and what is happening in this skyline simulator🙏
